### PR TITLE
feat: fullfill the page security.md

### PR DESCRIPTION
**Pull request by StakeandRelax-Validator**
Hello,
we try to add our contribution to setup with a bit of information the page security.md.
The informations are taken from evmos/cro/juno documentation with the adeguate corrections in name of binary and chain-id.
It could be improved further, but we think that the source documentations are well done, so it's a good point to start.
Hope you'll apreciate it
Wookie

### CODE REVIEW

**Comment by StakeandRelax-Validator** [comment]
Added commit lint

**Comment by StakeandRelax-Validator** [comment]
corrected all markdown errors

File: docs/nodes/kms.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<!--
+order: 2
+-->
+
+# Tendermint KMS
+
+Set up a Key Management System for okp4 {synopsis}
```
Comment (ad2ien) [inline docs/nodes/kms.md:7]: Do you mean something like this?
```suggestion
## Synopsis
Set up a Key Management System for okp4
```

File: docs/nodes/security.md [hunk 2]
```diff
@@ -2,18 +2,190 @@
...
+
+Validators are responsible for ensuring that the network can sustain denial of service attacks.
+
+One recommended way to mitigate these risks is for validators to carefully structure their network topology in a so-called sentry node architecture.
+
+Validator nodes should only connect to full-nodes they trust because they operate them themselves or are run by other validators they know socially. A validator node will typically run in a data center. Most data centers provide direct links the networks of major cloud providers. The validator can use those links to connect to sentry nodes in the cloud. This shifts the burden of denial-of-service from the validator's node directly to its sentry nodes, and may require new sentry nodes be spun up or activated to mitigate attacks on existing ones.
+
+Sentry nodes can be quickly spun up or change their IP addresses. Because the links to the sentry nodes are in private IP space, an internet based attacked cannot disturb them directly. This will ensure validator block proposals and votes always make it to the rest of the network.
+
+::: tip
```
Comment (ad2ien) [inline docs/nodes/security.md:167]:
```suggestion
:::tip
```

File: docs/nodes/kms_ledger.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+order: 3
+-->
+
+# Tendermint KMS + Ledger
+
+Set up Tendermint KMS with the Tendermint Ledger app {synopsis}
+
+## Pre-requisites
+
+- [Ledger device](https://shop.ledger.com/) {prereq}
```
Comment (ad2ien) [inline docs/nodes/kms_ledger.md:11]: Should `{prereq}` be interpreted in some way?
```suggestion
- [Ledger device](https://shop.ledger.com/)
```

**Review by ad2ien — CHANGES_REQUESTED** [review]
🤩 thanks a lot!
Just some remarks regarding the rendering so that your work looks nice as it should 😉

**Review by ad2ien — APPROVED** [review]
🚀

**Comment by StakeandRelax-Validator** [comment]
had to fix a bit of old commits subjects but now should be all correct. There is still commit c9ab9ba without lint, should I fix it also?

**Comment by ccamel** [comment]
@StakeandRelax-Validator Thank you very much for your input, we really appreciate it.

From what I see, the git commits history is not clean, but what I can suggest, if you agree, is that I squash & merge in one commit your work on main. This will solve the history problem as well as potentially the commit messages.

**Comment by ccamel** [comment]
@amimart need your approval.

**Review by amimart — APPROVED** [review]
That's a really great work, interesting and relevant, thank you so much for that! 🙏

**Review by ccamel — APPROVED** [review]
That's a really great piece of work. Thanks so much for your submission @StakeandRelax-Validator :sunglasses:

**Pull request merged**